src/main/java/frc/robot/commands/Score.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.elevator;
+import frc.robot.subsystems.subWristem;
+
+public class Score extends CommandBase {
+  static final double highHeight = 1462;
+  static final double midHeight = 1614;
+  elevator Elevator = new elevator();
+  subWristem intake = new subWristem();
+  double height;
+  /** Creates a new Score. */
+  public Score(int x) {
+    switch(x){
+      case 0: height = 0; //lowest height
+      break;
+      case 1: height = midHeight; //mid height
+      break;
+      case 2: height = highHeight; //highest height 
+      break;
+    }
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
```

Comment:
Should pass in the subsystems as arguments instead of initialising them directly and don't forget to use the addRequirements method